src/app/src/util/constants.js

```diff
@@ -0,0 +1,94 @@
+export const OTHER = 'Other';
+
+export const contributorTypeOptions = Object.freeze([
```

Comment:
That is the DRYest way to handle it, but adding complexity to accomplish it is not a requirement since:
  * The list only appears 2 places.
  * The list is hard coded and may never change. If it does change, it will be be an infrequent occurrence.

Reply:
Sounds good, I'll just add the comments then.

Reply:
See 
72de9c9  and 
296d0a0